### DISCUSSION TITLE
Uncaught TypeError: this.endComment is null on wire:model.live

### DISCUSSION THREAD

**Discussion by taqiussa** [livewire/livewire#6725] [7. Bugs (archived)]
### Livewire version

3.0.2

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Make normal livewire component
2. make normal table with search functionality and limit page on paginate
3. use wire:model.live="search" and wire:model.live="limit" on blade
4. use withpagination
5. normal query on laravel eloquent model

### Additional information

it throw Uncaught TypeError: this.endComment is null when changing select limit paginate, 
on search it throw Uncaught TypeError: this.endComment is null when deleting value after searching something

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment]
Hi there, I'm unable to replicate this issue. This is the component I've tested based on your steps:

```php
<?php

namespace App\Livewire;

use App\Models\User;
use Livewire\Attributes\Computed;
use Livewire\Component;
use Livewire\WithPagination;


class Users extends Component
{
    use WithPagination;

    public $search;
    public $limit = 2;

    #[Computed]
    public function users()
    {
        return User::query()
            ->when($this->search, fn($query) => $query->where('name', 'like', "%$this->search%"))
            ->paginate($this->limit);
    }

    public function render()
    {
        return <<<'BLADE'
            <div>
                <input type="text" wire:model.live="search">
                <input type="text" wire:model.live="limit">

                <div>
                    @foreach($this->users as $user)
                        <div wire:key="{{ $user->id }}">{{ $user->id }} {{ $user->name }}</div>
                    @endforeach
                </div>

                <div>
                    {{ $this->users->links() }}
                </div>
            </div>
        BLADE;
    }
}
```

**Reply by Shankar-Tom** [reply]
<div>
                <input type="text" wire:model.live="search">
                <input type="text" wire:model.live="limit">

                <div>
                    @foreach($this->users as $user)
                        <div wire:key="{{ $user->id }}">{{ $user->id }} {{ $user->name }}</div>
                    @endforeach
                </div>

                <div>
                    {{ $this->users->links() }}
                </div>
            </div>      put this into .blade.php   file then you will get the error

**Comment by TitanDvd** [comment]
I got the same error, but after of two hours of debugging based on test-error I get a solution.
The following code example will throw a livewire error on browser console: `Uncaught TypeError: this.endComment is null`

```
<input list = "myList" type="text" wire:model = "TheModel">
<datalist id = "myList">

           // PermissionsAndModules is a computed property
          @foreach ($this->PermissionsAndModules as $permAndMod)
               <option value = "{{$permAndMod}}">
          @endforeach

</datalist>
```

To fix it I just close the `option` tag like a `select` tag

```
<input list = "myList" type="text" wire:model = "TheModel">
<datalist id = "myList">

           // PermissionsAndModules is a computed property
          @foreach ($this->PermissionsAndModules as $permAndMod)
               <option value = "{{$permAndMod}}"></option>
          @endforeach

</datalist>
```

This is a particular solution for a particular case. I hope this help.
